source/docs/getting-started/making-selections.md

```diff
@@ -1,21 +1,25 @@
 # Making Selections
 
-The selection tool is also a useful feature to master when beginning to learn Avogadro. Generally, the selection tool allows for the individual selection of atoms, bonds, or fragments.
+The [Selection Tool](tools-selection-tool) is also a useful feature to master when beginning to learn Avogadro. Generally, the Selection Tool allows for the individual selection of atoms, bonds, or fragments.
 
 ![](../../_static/tutorial-select-tool.png)
 
-The select provides you with the ability to select a single atom within a molecule. This is achieved by left clicking the atom. Press and holding the "shift" key allows for the selection of multiple atoms.
+The Selection Tool provides you with the ability to select a single atom within a molecule. This is achieved by left-clicking the atom. Pressing and holding <kbd>shift</kbd> while clicking allows for the selection of multiple atoms.
 
-Right clicking on the black display will clear the selection made.
+Right-clicking on an empty part of the view will clear the selection made.
+
+```{warning}
+A known bug means that on some systems, multiple left-clicks are required to clear the selection.
```

Comment:
This should be fixed in 1.99, right?

Reply:
Hmm, good point, I haven't actually explicitly checked, but equally hadn't noticed it suddenly working correctly either. I'll have a look on Monday.

Reply:
I checked and it's not fixed, still behaves the same on my system.